ui/src/app/components/equivalent-gcloud-command/equivalent-gcloud-command.component.html

```diff
@@ -0,0 +1,12 @@
+<div mat-dialog-content>
+  <h2>Equivalent gcloud command line<mat-icon class="configure"
+    matTooltip='This is the gcloud command to launch the dataflow job with the same parameters.'>
+  info</mat-icon></h2>
+  
+  <span class="left-text">{{ gcloudCmd }}</span>
```

Comment:
Lets use something like - 
https://www.npmjs.com/package/ngx-highlightjs  to display code snippets in Harbourbridge.

Demo of the package - https://ngx-highlight.netlify.app/